src/PointwiseFunctions/Elasticity/ConstitutiveRelations/IsotropicHomogeneous.hpp

```diff
@@ -0,0 +1,104 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <limits>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Options/Options.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace Elasticity {
+/*!
+ * \brief Constitutive relations for elastic materials
+ */
+namespace ConstitutiveRelations {
+
+/*!
+ * \brief An isotropic and homogeneous material
+ *
+ * \details For an isotropic and homogeneous material the linear constitutive
+ * relation \f$T^{ij}=\chi^{ijkl}S_{kl}\f$ reduces to
+ * \f[
+ * \chi^{ijkl} = \lambda \delta^{ij}\delta^{kl} + \mu
+ * \left(\delta^{ik}\delta^{jl} + \delta^{il}\delta^{jk}\right) \\
+ * \implies \quad T^{ij} = \lambda \mathrm{Tr}(S) \delta^{ij} + 2\mu S^{ij}
+ * \f]
+ * with the _Lamé parameter_ \f$\lambda\f$ and the _rigidity_ (or _shear
+ * modulus_) \f$\mu\f$. In the parametrization chosen in this implementation we
+ * use the _incompressibility_ (or _bulk modulus_) \f[K=\lambda +
+ * \frac{2}{3}\mu\f] instead of the Lamé parameter. In this parametrization the
+ * stress-strain relation \f[ T^{ij} = K \mathrm{Tr}(S) \delta^{ij} +
+ * 2\mu\left(S^{ij} - \frac{1}{3}\mathrm{Tr}(S)\delta^{ij}\right)
```

Comment:
Looks OK except RHS has the wrong sign. Also, might be worth defining the symmetric-trace-free part of the strain, which is the shear tensor, as in Thorne and Blandford Eq. (11.18). This makes the equation much simpler.